agent/agent.go

```diff
@@ -149,6 +167,13 @@ func (a *orbAgent) Stop() {
 		}
 	}
 	a.client.Disconnect(250)
+	defer close(a.hbDone)
+	defer close(a.policyRequestSucceeded)
+	defer close(a.groupRequestSucceeded)
+}
+
+func (a *orbAgent) closeRequestTickers() {
+
```

Comment:
empty?

Reply:
I was using for testing, forgot to remove that.